ecobee/functions.go

```diff
@@ -82,6 +81,11 @@ func (c *Client) GetThermostat(thermostatID string) (*Thermostat, error) {
 		IncludeSensors:         true,
 		IncludeWeather:         false,
 	}
+
+	for _, optfn := range opts {
```

Comment:
just `o` is ok here.  Or `opt` if you'd prefer.  The scope is one line, there won't be any confusion.

I try to follow the [Google Go style guide](https://google.github.io/styleguide/go/)